rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/model/ParametricFileFacts.java

```diff
@@ -319,20 +319,25 @@ private void sendDiagnostics() {
                 logger.debug("Cannot send diagnostics since the client hasn't been registered yet");
                 return;
             }
-            var messages = Lists.union(
-                unwrap(parserDiagnostics),
-                unwrap(analyzerDiagnostics),
-                unwrap(builderDiagnostics));
-            logger.trace("Sending diagnostics for {}. {} messages", file, messages.size());
+
+            // Read the atomic references once for the whole method, to ensure
```

Comment:
why was unwrapped removed? now there are 2 `.get` function calls? (I mean, maybe we can find a better way for the unwrap function, or add a comment next to the unwrap)

Reply:
`unwrap` was removed because we have extended the `logger.trace` call with version information. The old `unwrap` method just dropped the version info.